README.md

```diff
@@ -2,6 +2,10 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+## Install create-react-app
+
+sudo npm -s install create-react-app
+
```

Comment:
Please add : npm install react-scripts --save 
To install scripts from react to run the app.

Reply:
Done.